patches/com_buildbarn_bb_storage/base_image.diff

```diff
@@ -29,4 +29,21 @@ index e4aa679..6e92429 100644
 +        base = Label("@distroless_base"),
          entrypoint = ["/app/{}".format(native.package_relative_label(binary).name)],
          tars = [tar_target],
-         # Don't build un-transitioned images, as the default target architecture might be unsupported
\ No newline at end of file
+         # Don't build un-transitioned images, as the default target architecture might be unsupported
+@@ -32,7 +32,7 @@ def multiarch_go_image(name, binary):
+
+     for arch in ["amd64", "arm64"]:
+         arch_image_target = "{}_{}_image".format(name, arch)
+-        target_platform = "@rules_go//go/toolchain:linux_" + arch
++        target_platform = "@rules_go//go/toolchain:linux_" + arch + "_cgo"
+         images.append(arch_image_target)
+         platform_transition_filegroup(
+             name = arch_image_target,
+@@ -49,6 +49,6 @@ def container_push_official(name, image, component):
+     oci_push(
+         name = name,
+         image = image,
+-        repository = "ghcr.io/buildbarn/" + component,
++        repository = "ghcr.io/buildbarn/" + component + "/backend",
```

Comment:
Do we need this? I would have thought `ghcr.io/buildbarn/bb_portal` and `ghcr.io/buildbarn/bb_portal_frontend` would suffice

Reply:
seems neater to me.  maybe we can change frontend to match as well, ...but if you want to adjust in a subsequent PR I'm totally fine w/that.